static/bidder-info/programmaticX.yaml

```diff
@@ -0,0 +1,12 @@
+aliasOf: 'teqblaze'
+endpoint: "https://us-east.progrtb.com/pserver"
+maintainer:
+  email: "pxteam@programmaticx.ai"
+gvlVendorID: 1344
+userSync:
+  redirect:
+    url: "https://sync.progrtb.com/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redirect={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
Is redirect enabled? I'm getting a 204 using this URL: `https://sync.progrtb.com/pbserver?gdpr=0&gdpr_consent=&ccpa=&gpp=&gpp_sid=&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3DprogrammaticX%26gdpr%3D0%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%5BUID%5D`

<img width="1030" height="137" alt="Screenshot 2025-08-28 at 2 26 59 PM" src="https://github.com/user-attachments/assets/3c827fd7-930e-4281-ba22-e85b3761ed43" />